Clients/src/presentation/components/Table/index.tsx

```
@@ -129,79 +129,95 @@ const BasicTable = ({
+            sx={{
+              ...singleTheme.tableStyles.primary.body.row, 
+              height: "36px", 
+              "&:hover":{
+                backgroundColor: "#FBFBFB",
+                cursor: "pointer",
+              }
+            }}
+            key={row.id}
+            onClick={() => {
+              console.log(`Row clicked: ${row.id}`);
+              onRowClick && onRowClick(row.id as number);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)